https://github.com/KDE/ghostwriter/commit/b76b14a00069a2d544b615ad6dc8f9cbdc678982

From b76b14a00069a2d544b615ad6dc8f9cbdc678982 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Wed, 26 Jul 2023 04:19:19 +0200
Subject: [PATCH] Include <cstdint> where needed

Include it where std::uint32_t is explicitly used.

(cherry picked from commit 9e1a2ba2e01c7172a04e41b5d487461aa73c87d6)
---
 src/markdownnode.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/markdownnode.cpp b/src/markdownnode.cpp
index dce179d4a..f7a4eab49 100644
--- a/src/markdownnode.cpp
+++ b/src/markdownnode.cpp
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: GPL-3.0-or-later
  */
 
+#include <cstdint>
+
 #include "../3rdparty/cmark-gfm/src/cmark-gfm.h"
 #include "../3rdparty/cmark-gfm/extensions/cmark-gfm-core-extensions.h"
 
